Doc/reference/compound_stmts.rst

```diff
@@ -302,8 +302,8 @@ stored in the :mod:`sys` module and can be accessed via :func:`sys.exc_info`.
 :func:`sys.exc_info` returns a 3-tuple consisting of the exception class, the
 exception instance and a traceback object (see section :ref:`types`) identifying
 the point in the program where the exception occurred.  :func:`sys.exc_info`
-values are restored to their previous values (before the call) when returning
-from a function that handled an exception.
+values are restored to their previous values (before the call) when leaving
```

Comment:
Is "before the call" correct now that we are no longer talking about a function call?

Reply:
I don't like "VALUES are restored to their previous VALUES".

How about "sys.exc_info ELEMENTS are restored to their previous VALUES"?

Reply:
@iritkatriel Thanks, I agree, I should have read more carefully. Fixed.